addons/payment/controllers/post_processing.py

```diff
@@ -58,11 +58,11 @@ def poll_status(self, **_kwargs):
         for tx in monitored_txs:
             display_message = None
             if tx.state == 'pending':
-                display_message = tx.provider_id.pending_msg
+                display_message = _(tx.provider_id.pending_msg)
```

Comment:
You cannot translate a term stored in a variable; you can only translate string primitives. The message should already be translated at this point.